### DISCUSSION TITLE
congratulation to Sandboxes (Jail-like Containers) // jailmaker soon obsolete

### DISCUSSION THREAD

**Discussion by PackElend** [Jip-Hop/jailmaker#79] [Q&A] [answered]
Hi,
seems you have made it: https://www.truenas.com/docs/scale/scaletutorials/apps/sandboxes/ ❤️ 
Will you stop working on jailmaker?

thx

**Comment by Jip-Hop** [comment, marked as answer]
Thanks yes I will continue working on it as jailmaker itself is not included with SCALE. Only the systemd-nspawn dependency (which is of course still good news).

Have a look at this:
https://www.truenas.com/docs/scale/scaletutorials/apps/sandboxes/

**Comment by neoKushan** [comment]
I came here to see if this had been discussed as well. When I adopted TrueNAS last year, I really struggled with the app ecosystem and yearned for a solution that just let me `docker compose` my way to success. There were several terrible solutions to this (Like running a VM), but Jailmaker was ideal and has been running completely without issue for me. 

I was worried that iX would do something in a future version that prevented its use, but it seems they've realised that your solution plugs a very real gap and makes a lot of sense. So thank you for the script/tool and thank you for maintaining it well enough that iX is giving it somewhat more than an unofficial blessing.

**Comment by Jorsher** [comment]
I've been grateful for this tool. Very useful if I want to run a package or scripts on the host in its own little container. Losing Jails was my biggest downside for moving to Scale from Core. Like the previous poster, I was worried iX would eventually prevent this from being an option, so I was happy to see it sorta-supported.

Thanks again!
